lib/block_view.go

```diff
@@ -742,6 +609,9 @@ type UtxoView struct {
 	// Messages data
 	MessageKeyToMessageEntry map[MessageKey]*MessageEntry
 
+	// Postgres stores message data slightly differently
+	MessageMap map[BlockHash]*Message
```

Comment:
Not using overloaded terms like "message" as variable names is preferred (same reasoning as not using single letter variable names).  I'd suggest we call this new type of message entry a, "PGMessageEntry" instead. And "PGMessageEntryMap".  If we ever rip out badger, we can then easily find and replace and kill all the PG tags.

Reply:
I also think this naming convention will help to clarify the distinction between the two message types for new developers working on the code base.